blog/2024-09-17-wasmcloud-postgres-capability-brings-pluggable-databases/index.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: "The wasmCloud Postgres capability brings pluggable databases for your apps"
+authors: [ericgregory]
+image: './images/wasmcloud-postgres-capability.webp'
+slug: 2024-09-17-wasmcloud-postgres-capability-brings-pluggable-databases
+date: "2024-09-17"
+description: "Learn how to get started with the Postgres capability in wasmCloud."
+---
+
+![The wasmCloud Postgres capability brings pluggable databases for your apps](./images/wasmcloud-postgres-capability.webp)
+
+With wasmCloud, you can build applications out of WebAssembly components and pluggable, reusable [**capabilities**](/docs/concepts/capabilities), so you can focus on the core logic of your application and abstract away the rest, all using open source standards.
+
+If you want to use a database system like **Postgres**, wasmCloud makes it easy to simply plug in the functionality in a reusable, language-agnostic way, without wasting time writing boilerplate code. In this post, we'll show you how to get started with the Postgres capability in wasmCloud.
```

Comment:
A sentence like "Postgres is one of the most used relational database systems", and Postgres should probably be a link to https://postgresql.org, as this mention is earlier